### DISCUSSION TITLE
Proposition: lapce like sticky header

### DISCUSSION THREAD

**Discussion by oknozor** [zed-industries/zed#11721] [Feature Requests]
Hello there, 
I started using zed recently and I really enjoy it so far. I would be curious to discover the codebase and contribute. 

One thing I really like in the Lapce editor is sticky header, this is really convenient when editing large function to remember where you are. 

![image](https://github.com/zed-industries/zed/assets/24563836/4c3b8bce-7143-4986-b5c0-a88e02dff1d8)

Would that be something you would like to see in Zed ? 
If yes I'd be happy to start hacking on the editor implementing this feature.

**Comment by Moshyfawn** [comment]
This has already been proposed in #5344. Be sure to +👍 the feature request.

**Reply by oknozor** [reply]
Done thank you
